source/fundamentals/connection.txt

```diff
@@ -245,6 +245,16 @@ that Context deadline for the operation. If the context does not have a
 deadline, the driver derives a new Context from the given Context using
 the ``Timeout`` option set on the ``Client``.
 
+.. note:: Retryability
```

Comment:
Issue: 

I think the note title "Retryability" doesn't match the message and that the message introduces two different things, but should maybe only focus on one. 

To me, the term retryability describes how easily you can try to deliver a command or message that failed by some measure. Although the term is used as a header in the spec, I would avoid using that term to introduce this information.

The two separate topics I'm reading are:
1) "starting in (version), retryable reads and writes are enabled by default"
2) "When retryable reads/writes are enabled and the timeout option is set, there's a specific behavior"

Suggestion:
Since the reader may have already enabled retryable reads or writes, describe the conditions that lead to the retry behavior first. E.g.

"Starting in (version), when you enable retryable reads or writes and set the Timeout option in your Client, the driver (describe retry behavior)"

And then the final sentence (or possibly somewhere else) mention the other piece of information, e.g.:
"Retryable reads and writes are enabled by default starting in (version)"

Reply:
The user does not have to enable retryable reads or writes, this behavior is enabled by default for nearly all versions of the Go driver. I removed the title and rearranged the sentences.

Reply:
Oh ok, that part wasn't clear to me (that the default values are enabled prior to this version).
I think it still might be important to mention the initial conditions since they might not be using the default settings for reads/writes, but will leave that up to you.